test/Core.Test/AdminConsole/OrganizationFeatures/OrganizationUsers/AcceptOrgUserCommandTests.cs

```diff
@@ -167,7 +169,7 @@ public async Task AcceptOrgUserAsync_UserWithout2FAJoining2FARequiredOrg_ThrowsB
         SetupCommonAcceptOrgUserMocks(sutProvider, user, org, orgUser, adminUserDetails);
 
         // User doesn't have 2FA enabled
-        _twoFactorIsEnabledQuery.TwoFactorIsEnabledAsync(user).Returns(false);
```

Comment:
Is it possible to delete this class property altogether? I can't see that it does anything because it's not actually injected into the sut.

Reply:
It is used to check the user's 2FA. I removed redundant setups as this is already done in `SetupCommonAcceptOrgUserMocks`